src/crypto/openssl/rsa_public_key.cpp

```diff
@@ -208,6 +208,22 @@ namespace ccf::crypto
              pctx, signature, signature_size, hash.data(), hash.size()) == 1;
   }
 
+  bool RSAPublicKey_OpenSSL::verify_pkcs1(
+    const uint8_t* contents,
+    size_t contents_size,
+    const uint8_t* signature,
+    size_t signature_size,
+    MDType md_type)
+  {
+    auto hash = OpenSSLHashProvider().Hash(contents, contents_size, md_type);
+    Unique_EVP_PKEY_CTX pctx(key);
+    CHECK1(EVP_PKEY_verify_init(pctx));
+    CHECK1(EVP_PKEY_CTX_set_rsa_padding(pctx, RSA_PKCS1_PADDING));
```

Comment:
Still impressed by how you find the padding inconsistencies! So is `RSA_PKCS1_PADDING` referring to the `RSASSA-PKCS1-v1_5` that you mentioned in the github issue https://github.com/microsoft/CCF/issues/6601#issuecomment-2512059875?

Reply:
> Still impressed by how you find the padding inconsistencies! 

Because token verification doesn't pass? :)

> So is RSA_PKCS1_PADDING referring to the RSASSA-PKCS1-v1_5 that you mentioned in the github issue

Yes, it is